package.json

```diff
@@ -123,4 +123,3 @@
     }
   }
 }
-
```

Comment:
Please revert the changes in the package and package lock.

Reply:
This has been reverted